pkg/status/handlers_test.go

```diff
@@ -122,3 +122,34 @@ func TestHealthFailure(t *testing.T) {
 		t.Fatalf("expected Hydra to be false not  %v", receivedStatus.Hydra)
 	}
 }
+
+func TestDeploymentInfo(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+
+	mockLogger := NewMockLoggerInterface(ctrl)
+	mockMonitor := NewMockMonitorInterface(ctrl)
+	mockTracer := NewMockTracingInterface(ctrl)
+	mockService := NewMockServiceInterface(ctrl)
+
+	req := httptest.NewRequest(http.MethodGet, "/api/v0/info", nil)
+	w := httptest.NewRecorder()
+
+	mux := chi.NewMux()
+	NewAPI(false, mockService, mockTracer, mockMonitor, mockLogger).RegisterEndpoints(mux)
+
+	mux.ServeHTTP(w, req)
```

Comment:
can we switch to the "essential" unit test, relying on the specific endpoint instead of instantiating a Mux ?